src/app/shared/services/api/search.repo.ts

```diff
@@ -62,18 +63,18 @@ export class SearchRepo extends BaseRepo {
     tags: any[] = [],
     archiveId: string,
     limit?: number,
-  ): Observable<SearchResponse> {
+  ) {
     const data = {
       query,
       tags:'',
       archiveId,
       publicOnly:true
     };
 
-    return this.http.getV2Request<SearchResponse>(
+    return getFirst(this.httpV2.get<any>(
```

Comment:
Can you avoid using the `any` type here? Also, can  you update the call to `httpV2.get` to include an options parameter to disable passing in the Auth Token? It would look something like this:
```js
this.httpV2.get(
  '/search/folderAndRecord',
  data,
  null,
  {
    authToken: false
  }
)
```

This makes sure that if the `AuthToken` used in certain V2 requests is defined, it won't be passed into the request. Since I was working on Directives stuff earlier, I had the `AuthToken` defined and this specific endpoint will respond with a 401 error if you pass in an authorization token.